content/embeds/rc-version-upgrade.md

```diff
@@ -0,0 +1,11 @@
+Select **Version upgrade** to request to upgrade your subscription and databases (*Redis 7.2 and earlier only*).
```

Comment:
I wouldn't mention specific versions. It will force us to update in the future. Just saying later available version is enough IMO

Reply:
```suggestion
Select **Version upgrade** to request to upgrade your subscription and databases if there is a later version available.
```